tests/formats/csv/nested.yaml

```
@@ -0,0 +1,11 @@
+zql: '*'
+
+input: |
+        #0:record[a:string,b:record[c:string,d:string]]
+        0:[hello;[world;goodbye;]]
+
+output-format: csv
+
+output: |
+        a,b.c,b.d
+        hello,world,goodbye
```

Elsewhere we indent two spaces per level in YAML.
```suggestion
  #0:record[a:string,b:record[c:string,d:string]]
  0:[hello;[world;goodbye;]]

output-format: csv

output: |
  a,b.c,b.d
  hello,world,goodbye
```